org.abapgit.adt.ui/src/org/abapgit/adt/ui/internal/repositories/wizards/AbapGitWizardPageBranchSelection.java

```diff
@@ -0,0 +1,27 @@
+package org.abapgit.adt.ui.internal.repositories.wizards;
+
+import org.abapgit.adt.ui.internal.wizards.AbapGitWizard.CloneData;
+import org.abapgit.adt.ui.internal.wizards.AbapGitWizardPageBranchAndPackage;
+import org.eclipse.core.resources.IProject;
+
+public class AbapGitWizardPageBranchSelection extends AbapGitWizardPageBranchAndPackage {
+
+	public AbapGitWizardPageBranchSelection(IProject project, String destination, CloneData cloneData, Boolean pullAction) {
+		super(project, destination, cloneData, pullAction);
+	}
```

Comment:
Also set a new title & description for the page. The parent page sets the title "Branch and Package Selection", but the branch selection wizard does not allow to select package.
Hence a new title & description should be set here.

Suggestion.
Title: Select Branch (Same as the credentials page)
Description: Select branch to switch from the dropdown